ckanext/geocat/tests/conftest.py

```diff
@@ -1,4 +1,28 @@
+import json
+import os.path
+
 import pytest
+from ckan.tests import factories
+
+from ckanext.harvest.tests import factories as harvest_factories
+
+location = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+mock_capabilities_url = (
+    "http://mock-geocat.ch/?version=2.0.2&request=GetCapabilities&service=CSW"
+)
+
+
+@pytest.fixture
+def ogdch_requests_mock(requests_mock):
```

Comment:
There is a `requests_mock` fixture already registered with pytest, so we don't need to import it separately or add it to our test methods with a decorator any more - see https://requests-mock.readthedocs.io/en/latest/pytest.html for more details.

This fixture, `ogdch_requests_mock`, is used to set up the `requests_mock` object with things we always want to be true.

`requests_mock.real_http = True` (below) means that if our code calls a url that has not been registered with the mock object, it will really be called, instead of returning an error.